shared_model/backend/protobuf/impl/proto_query_response_factory.cpp

```diff
@@ -68,6 +68,29 @@ shared_model::proto::ProtoQueryResponseFactory::createAccountAssetResponse(
       query_hash);
 }
 
+std::unique_ptr<shared_model::interface::QueryResponse>
+shared_model::proto::ProtoQueryResponseFactory::createAccountAssetResponse(
+    const std::vector<interface::types::AccountIdType> &account_ids,
```

Comment:
Better accept these arguments in touched functions by value, otherwise moving them here will make no sense